src/scss/base/_variables.scss

```diff
@@ -14,8 +14,14 @@ $gris: hsl(213, 9%, 39%);
 //Font
 $fuentePrincipal: 'Barlow', sans-serif;
 $fuenteSecundaria: 'Fraunces', serif;
+$fuenteTitulo: 'Lato', sans-serif;
 
 //Font Gruesor
 $black: 900;
 $bold: 700;
-$regular: 600;
\ No newline at end of file
+$regular: 600;
+
+//Media Queries
+$telefono:360px;
```

Comment:
change to phone, and add space between ":" and the value, to have something like this: **$phone: 360px;**